api/rest/stat_all.md

```diff
@@ -10,17 +10,18 @@ grand_parent: API
 
 Returns a multiple level object containing all available time series for the specified
 places and statistical variables.
-See the [full list of StatisticalVariables](/statistical_variables.html).
+Note that in Data Commons, a Statistical Variable is any type of statistical metric that can be measured at a place and
+time. See the [full list of StatisticalVariables](/statistical_variables.html).
 
 **URL**: `/stat/all`
 
 **Method**: `POST`
 
 **Required Arguments**:
 
-- `places`: A list of [`Place`](https://datacommons.org/browser/Place) `dcid`s to query for.
+- `places`: A list of [`Place`](https://datacommons.org/browser/Place) `DCIDs` to query for. (Here DCID stands for Data Commons ID, the unique identifier assigned to all entities in Data Commons.)
```

Comment:
Do you want to link to the glossary?

Reply:
I'm not really sure the right call here.

Reply:
I wasn't going to link to the glossary, since it is not published at this time. That having been said, I can definitely refer readers there if you prefer!